src/main/resources/db/migration/V6__INTRODUCE_UUID_DATA_TYPE.sql

```diff
@@ -0,0 +1,21 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+ALTER TABLE RESOURCE
+    ADD COLUMN UUID_ID UUID;
+
+ALTER TABLE RESOURCE
+    DROP CONSTRAINT RESOURCE_pkey;
+
+ALTER TABLE RESOURCE
+    RENAME COLUMN ID TO OLD_ID;
+
+UPDATE RESOURCE SET UUID_ID = uuid_generate_v1();
+
+ALTER TABLE RESOURCE
+    RENAME COLUMN UUID_ID TO ID;
+
+ALTER TABLE RESOURCE
+    ADD PRIMARY KEY (ID);
+
+ALTER TABLE RESOURCE
+    DROP COLUMN OLD_ID;
```

Comment:
Is it maybe easier to simply drop the table, then recreate it?

Reply:
And that's also the reason why the UUID extension is definitely needed (as it stands now), because I'm creating UUIDs for potentially existing data entries during this migration. After that the DB is no longer in charge of UUID creation.